InfiniteRecharge/src/main/java/frc/robot/subsystem/BitBucketSubsystem.java

```diff
@@ -107,4 +109,6 @@ public void initialize() {
     public abstract void dashboardPeriodic(float deltaTime);
 
     public abstract void disable();
+
+    public abstract BaseTalon[] getTalons();
```

Comment:
This should really be a List rather than an array. It makes iterating over it easier and lets you take advantage of java streams and other fanciness.